src/Aspire.Hosting.Azure.ContainerApps/AzureContanierAppsInfrastructure.cs

```diff
@@ -127,6 +128,26 @@ record struct EndpointMapping(string Scheme, string Host, int Port, int? TargetP
 
             public List<(ContainerAppVolume, ContainerAppVolumeMount)> Volumes { get; } = [];
 
+            public void WriteToManifest(ManifestPublishingContext context, AzureConstructResource construct)
+            {
+                construct.WriteToManifest(context);
+
+                // We're handling custom resource writing here instead of in the AzureConstructResource
+                // this is because we're tracking the BicepParameter instances as we process the resource
+                if (Parameters.Count > 0)
+                {
+                    context.Writer.WriteStartObject("params");
```

Comment:
Could we end up in a situation where `AzureConstructResource` wrote the `params` object, and then now we are writing another one to the same object?

Reply:
That would be a bug in our code. Code outside never gets access to the `AzureConstructResource` so its not a concern.

Reply:
Done